frontend/src/components/Nav/Nav.jsx

```diff
@@ -3,12 +3,14 @@ import NavSearchBar from './NavSearchBar'
 
 export default function Nav() {
   return (
-    <div>
-      <div>
-        <img src="" alt="Logo" srcset="" />
-        <div>Title</div>
+    <nav className='w-full flex justify-center  py-8 bg-cyan-50'>
+      <div className='w-7/12 flex justify-center bg-teal-200'>
+        <div className='flex justify-center items-center mr-auto px-12 bg-yellow-300 '>
+          <img src="" alt="Logo" srcset="" />
+          <div>Title</div>
```

Comment:
So I'm just talking from a general perspective but I think we can make this nav component more "re-usable", are you able to pass in a string value called title? and then just insert the string here instead? That way we don't have a static "title" value.

Reply:
For example something like this? I'm not sure how this would work in react 
```
export default function Nav( string title)
```

and then in the div

```
<div>{title variable}</div>
```

Reply:
> 

That's correct. I wanted to position everything first then I'll work on that.